Mods/DiscordRichPresence/CMakeLists.txt

```diff
@@ -14,6 +28,7 @@ add_library(DiscordRpc SHARED
 
 target_include_directories(DiscordRpc PRIVATE
 	${CMAKE_CURRENT_SOURCE_DIR}/Src
+	${CMAKE_SOURCE_DIR}/External/DiscordSDK/cpp
 )
 
 target_link_libraries(DiscordRpc PRIVATE
```

Comment:
@OrfeasZ Here's the CMakeLists changes I was referring to in Discord.  A couple of my own comments:

1. I'm not sure if this is the best practice for using `ExternalProject_Add`, so your insight would be appreciated as I'd like to use this the right way 😀 
2. I've also noticed that sometimes VS just doesn't want to download the SDK when building... not sure if there's a surefire way to ensure that it gets downloaded or not.
3. The build is failing due to a linker error, which I'm assuming is due to me not putting anything down here in `target_link_libraries`.  However, I wasn't 100% sure what to put down here. I tried half-assing it by putting `${CMAKE_SOURCE_DIR}/External/DiscordSDK/lib/x86_64/discord_game_sdk.dll.lib` down here, but it didn't result to anything useful.